lib/rialto/etl/configs/stanford_grants_to_sparql_statements.rb

```diff
@@ -57,25 +57,33 @@
 
 # See https://wiki.duraspace.org/display/VTDA/VIVO-ISF+1.6+relationship+diagrams%3A+Grant
 to_field VIVO.relates.to_s, lambda { |json, accum|
-  person = RIALTO_PEOPLE[json['piEmployeeId']]
-  pi_role = RIALTO_CONTEXT_ROLES["#{json['spoNumber']}_#{json['piEmployeeId']}"]
+  family_name, given_name = json.fetch('piFullName', '').split(', ')
+  # Use `#to_s` explicitly here in case piFullName isn't a name string like we expect
+  person = Rialto::Etl::Transformers::People.resolve_person(given_name: given_name.to_s,
+                                                            family_name: family_name.to_s,
+                                                            addl_params: { sunetid: json['piSunetId'] })
+  # Do not link the grant to a person if a person could not be resolved
```

Comment:
Should log as a warning. (Note that ETL is configured to that warnings are written to a separate file, presumably so that it can be monitored for these sorts of anomalies.

Other than that, good to go.

Reply:
Done.